test/integration/WovnIndexSampleApacheTest.php

```diff
@@ -0,0 +1,225 @@
+JSON;
+        $expected = <<<EXPECTED
+<html lang="en"><head><script src="//j.wovn.io/1" async="true" data-wovnio="key=zwBmtA&amp;backend=true&amp;currentLang=ja&amp;defaultLang=ja&amp;urlPattern=path&amp;langCodeAliases={}&amp;langParamName=wovn&amp;version=0.0.1"> </script><link rel="alternate" hreflang="en" href="http://localhost/en/index.php"><link rel="alternate" hreflang="ja" href="http://localhost/index.php"></head><body>test</body></html>
+EXPECTED;
+
+        $this->writeFile('index.php', $index_php_content);
+
+        $this->setWovnIni($this->getWovnIni());
+        $this->setMockApiResponse($mock_api_response);
+
+        $response = $this->fetchURL('/index.php?wovn=ja');
+
+        $this->assertEquals($expected, $response->body);
+    }
+
+    public function testCheckAmpOption()
+    {
+        $amp_php_content = <<<CONTENT
+<?php require_once('WOVN.php/src/wovn_interceptor.php'); ?>
+<html ⚡><head></head><body>test</body></html>
+CONTENT;
+
+        $expected = <<<EXPECTED
+<html ⚡><head></head><body>test</body></html>
+EXPECTED;
+
+        $this->writeFile('amp.php', $amp_php_content);
+        $this->setWovnIni($this->getWovnIni(array('check_amp' => 1)));
+
+        $response = $this->fetchURL('/amp.php');
+
+        $this->assertEquals($expected, $response->body);
+    }
+
+    public function testStaticHtml()
+    {
+        $static_html_content = <<<CONTENT
+<html>
+  <head></head>
+  <body>
+    <h1>Static Content</h1>
+  </body>
+</html>
+CONTENT;
+
+        $expected = <<<EXPECTED
+<html>
+  <head><link rel="alternate" hreflang="ja" href="http://localhost/static.html?a=b&amp;wovn=ja"><link rel="alternate" hreflang="fr" href="http://localhost/static.html?a=b&amp;wovn=fr"><link rel="alternate" hreflang="bg" href="http://localhost/static.html?a=b&amp;wovn=bg"><link rel="alternate" hreflang="en" href="http://localhost/static.html?a=b"><script src="//j.wovn.io/1" data-wovnio="key=Tek3n&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=query&amp;langCodeAliases=[]&amp;langParamName=wovn" data-wovnio-info="version=WOVN.php_VERSION" async></script></head>
+  <body>
+    <h1>Static Content</h1>
+  </body>
+</html>
+EXPECTED;
+
+        $this->writeFile('static.html', $static_html_content);
+        $this->setWovnIni($this->getWovnIni());
+
+        $response = $this->fetchURL('/static.html?a=b');
+
+        $this->assertEquals($expected, $response->body);
+    }
+
+    private function fetchURL($path)
+    {
+        return Utils::fetchURL('http://localhost' . $path);
+    }
+
+    private function setMockApiResponse($contents)
+    {
+        $this->writeFile('v0/translation', $contents);
+    }
+
+    private function setWovnIni($contents)
+    {
+        $this->writeFile('wovn.ini', $contents);
+    }
+
+    private function setHtaccess($contents)
+    {
+        $this->writeFile('.htaccess', $contents);
+    }
+
+    private function writeFile($file, $contents)
```

Comment:
How about define writeFile at Utils.php.

Reply:
How can I manage the path of the document root?
I have a feeling that the code would be redundant, as shown below.
```
Utils::writeFile($this->docRoot . ‘/‘ . ‘wovn.ini’, $content);
Utils::writeFile($this->docRoot . ‘/‘ . ’static.html’, $content2);
```

Reply:
I prefer `Utils::writeFile($this->docRoot . ‘/‘ . ‘wovn.ini’, $content);` .
I can easily understand where is the file path.